shared/src/vyxal/Elements.scala

```diff
@@ -2386,10 +2589,130 @@ object Elements:
         VList.from(ListHelpers.map(a, iter).vzip(iter))
       case (a, b) =>
         ListHelpers.makeIterable(a).vzip(ListHelpers.makeIterable(b))
+    },
+    addDirect(
+      "£",
+      "Set Register",
+      List("set-register", "->register", "set-reg", "->reg"),
+      Some(1),
+      "a: any -> register = a",
+    ) { ctx ?=>
+      ctx.globals.register = ctx.pop()
+    },
+    addDirect(
+      "¥",
+      "Get Register",
+      List("get-register", "get-reg", "register", "<-register", "<-reg"),
+      None,
+      " -> push the value of the register",
+    ) { ctx ?=>
+      ctx.push(ctx.globals.register)
+    },
+    addDirect(
+      "←",
+      "Rotate Stack Left",
+      List("rotate-stack-left"),
+      None,
+      " -> rotate the entire stack left once",
+    ) { ctx ?=>
+      ctx.rotateLeft
+    },
+    addDirect(
+      "→",
+      "Rotate Stack Right",
+      List("rotate-stack-right"),
+      None,
+      " -> rotate the entire stack right once",
+    ) { ctx ?=>
+      ctx.rotateRight
+    },
+    addDirect(
+      "\\",
+      "Dump",
+      List("dump"),
+      Some(1),
+      "a: any -> dump all values on the stack",
+    ) { ctx ?=>
+      val a = ListHelpers.makeIterable(ctx.pop())
+      for v <- a do ctx.push(v)
```

Comment:
```suggestion
      ListHelpers.makeIterable(ctx.pop()).foreach(ctx.push)
```
Why type lot token when few token do trick